app/services/persona_detail_extraction.py

```
@@ -0,0 +1,70 @@
+import openai
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)